private/views/submitAuthors.hjs

```diff
@@ -9,54 +9,12 @@
 
   <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/bootswatch/3.3.4/cosmo/bootstrap.min.css">
   <link rel="stylesheet" href="http://maxcdn.bootstrapcdn.com/font-awesome/4.3.0/css/font-awesome.min.css">
-  <link rel="stylesheet" href="../../stylesheets/style.css">
+  <link rel="stylesheet" href="../stylesheets/style.css">
```

Comment:
/submitAuthors has the URL path /authors/submit. This means two steps behind: ../../